wsidicomizer/openslide.py

```diff
@@ -219,12 +218,18 @@ def __init__(
         )
         self._pyramid_index = int(math.log2(self.downsample))
 
-        base_mpp_x = float(self._slide.properties['openslide.mpp-x'])
-        base_mpp_y = float(self._slide.properties['openslide.mpp-y'])
-        self._pixel_spacing = SizeMm(
-            base_mpp_x * self.downsample / 1000.0,
-            base_mpp_y * self.downsample / 1000.0
-        )
+        try:
+            base_mpp_x = float(self._slide.properties['openslide.mpp-x'])
+            base_mpp_y = float(self._slide.properties['openslide.mpp-y'])
+            self._pixel_spacing = SizeMm(
+                base_mpp_x * self.downsample / 1000.0,
+                base_mpp_y * self.downsample / 1000.0
+            )
+        except KeyError:
+            raise Exception(
+                "Could not determine pixel spacing as openslide properties "
+                "'openslide.mpp-x' and/or 'openslide.mpp-y' is missing in file"
```

Comment:
`openslide.*` properties are generated by openslide, so if the user looks for those properties in their tiff-file they will never find it. So a more informative message would be something like:
- "Could not determine pixel spacing, openslide was unable to find or parse mpp from the file".

Reply:
Good comment. Changed the exception message.